swan-cern/files/swan_config_cern.py

```diff
@@ -42,6 +45,18 @@ async def get_swan_user_pod(self):
 
         return self.pod
 
+    def _check_tn_enabled(self):
+        user_roles = self.spawner.user_roles
+        tn_enabled = get_config('hub.config.tn.enabled', False)
+
+        if tn_enabled == True and "swan-ats" not in user_roles:
+           raise ValueError(
+                """
+                Access to the Technical Network cluster is not granted. 
+                Please <a href="https://cern.service-now.com/service-portal?id=sc_cat_item&name=request&se=Technical-Network" target="_blank">request access</a>
+                """)
+        return True
```

Comment:
You don't need to return anything, since you don't check the return value in the place you do the call.